# PcFeatures for a single cluster

Header header

geometry_msgs/Transform transform

geometry_msgs/Vector3 bb_center
sensor_msgs/PointCloud2 segmented_pc
